All Core Devs Meetings/Meeting 58.md

```diff
@@ -0,0 +1,298 @@
+# Ethereum Core Devs Meeting 58 Notes
+### Meeting Date/Time: Friday, March 29, 2019 14:00 UTC
+### Meeting Duration:  1 hours 45 minutes
+### [GitHub Agenda Page](https://github.com/ethereum/pm/issues/89)
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=v8Psbo8zY4Y)
+### Moderator: Hudson Jameson
+
+# Summary
+
+### DECISIONS MADE
+
+**DECISION 58.1** The [ProgPoW Carbon Vote](http://www.progpowcarbonvote.com/) signal will be shutting down in 13 days from today. Everyone that wishes to vote will need to do so before this date and ensure they leave their ETH in the address they voted from until after the specified block number.
+
+**DECISION 58.2** Going forward clients can provide an update in the comments in the agenda. If there are any questions or anything specific to discuss a space will be left to do so.
+
+**DECISION 58.3** Going forward Reseach can provide an update in the comments in the agenda. If there are any questions or anything specific to discuss a space will be left to do so.
+
+### ACTIONS REQUIRED
+
+**ACTION 58.1** Cat Herders to look at updating EIP1.
```

Comment:
Maybe add the GH issue link for current work: https://github.com/ethereum-cat-herders/PM/issues/19

Reply:
Updated.